src/core/viz/expressions/torque.js

```diff
@@ -203,10 +210,99 @@ export class Torque extends BaseExpression {
         return new Date(tmix);
     }
 
+    /**
+     * Set the time stamp of the simulation
+     * @api
+     * @memberof carto.expressions.Torque
+     * @instance
+     * @name getSimTime
+     * @param {Date} simulationTime - A javascript Date object with the new simulation time
+     */
+    setSimTime(simulationTime) {
+        const tmin = this._input.min.eval();
+        const tmax = this._input.max.eval();
+        if (!isDate(simulationTime)) {
+            throw new TypeError(`torque.setSimTime requires a valid date as parameter but got ${simulationTime}`);
+        }
+        if (simulationTime.getTime() < tmin) {
+            throw new RangeError('torque.setSimTime requires the date parameter to be higher than the minimal date of the dataset');
```

Comment:
It is not the minimum date of the dataset, that's the default behavior, but it can be overridden with `linear`